cylc/flow/etc/tutorial/runtime-tutorial/flow.cylc

```diff
@@ -34,10 +34,10 @@
         """
 
 [runtime]
-    [[get_observations_camborne]]
     [[get_observations_heathrow]]
-    [[get_observations_aldergrove]]
+    [[get_observations_camborne]]
     [[get_observations_shetland]]
+    [[get_observations_aldergrove]]
```

Comment:
Random order change?

Reply:
No, changed to match the tutorial

Reply:
These sections don't serve any purpose (they're empty).

They weren't present in the original tutorial:

https://github.com/metomi/rose/blob/2019.01.x/etc/tutorial/runtime-tutorial/suite.rc

I think they were randomly added when the tutorials were copied over, possibly just to make them validate for the purpose of tests. Suggest reverting to the way it was before and adapting the tests.

Reply:
I couldn't get the validation tasks to fail when I took them out... so easy!